content/docs/studio/user-guide/experiments/explore-ml-experiments.md

```diff
@@ -117,7 +123,14 @@ There are two types of filters:
 
     ![Result of commits on branch filter](https://static.iterative.ai/img/studio/commits_on_branch_filter.png)
 
-    </admon>
+    </details>
+
+  - In addition, you can also toggle between the `Custom filters` and apply one
```

Comment:
```suggestion
Customer filters can be un-applied without deleting. So that you can create the filters once and turn them on and off as many times as you need to.
```